ci/cleanup/aws.py

```diff
@@ -125,6 +123,28 @@ def clean_up_iam() -> None:
             print(f"Skipping role {role_name}")
 
 
+def get_testdrive_roles(client: Any) -> list[Any]:
+    roles = []
+    marker = None
+
+    while True:
+        if marker is not None:
+            response = client.list_roles(MaxItems=1000, Marker=marker)
+        else:
+            response = client.list_roles(MaxItems=1000)
```

Comment:
Why not just this?
```suggestion
        response = client.list_roles(MaxItems=1000, Marker=marker)
```

Reply:
It did not work due to:

```
botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid type for parameter Marker, value: None, type: <class 'NoneType'>, valid types: <class 'str'>
```

Reply:
And an empty parameter value neither did:
```
botocore.exceptions.ParamValidationError: Parameter validation failed: Invalid length for parameter Marker, value: 0, valid min length: 1
```